.github/workflows/build.yml

```diff
@@ -6,8 +6,16 @@ on:
       - master
       - ci
       - 'release/**'
+    paths-ignore:
+      - 'DOCS/**'
+      - 'etc/**'
```

Comment:
etc has a meson file and stuff that affects that final executable, not sure if you want to ignore those.

Reply:
Hmm yeah guess I'll drop that directory for now.